doc/tutorial/single_cell_detailed.rst

```diff
@@ -28,31 +33,12 @@ geometry and dynamics which can be constructed from 3 components:
    The decor also includes hints about how the cell is to be modelled under the hood, by
    splitting it into discrete control volumes (CV).
 
-Next, we construct a :class:`arbor.single_cell_model`. This model takes care of a lot of details
```

Comment:
Why was this removed?

Reply:
Because the introduction of the `single_cell_model` is in `single_cell_model.rst`, and we wanted some of the duplication that exists between the tutorials.